Implement profile page - GET /msadmins/me

piouson opened the pull request:
- [x] store profile data during login
- [x] edit data provider to handle profile request
- [x] process profile data
- [x] implement profile page `GET` view
- [x] change default profile page to show (previously edit)
- [x] remove profile edit page from view
- [x] remove edit button from profile show page
- [x] remove password field
- [x] add profile button to profile image link
- [ ] implement profile page `EDIT` view

Resolve #34

piouson committed:
feat: add initial profile page implementation

- [x] store profile data during login
- [x] edit data provider to handle profile request
- [x] process profile data
- [ ] implement profile page `GET` view
- [ ] implement profile page `EDIT` view
- [ ] add profile button to profile image link

Resolve #34

piouson committed:
Merge remote-tracking branch 'upstream/develop' into profile-page

piouson committed:
Merge remote-tracking branch 'upstream/develop' into profile-page

piouson committed:
feat: add profile view page

- [x] add profile view page
- [x] add profile link as user menu item
- [x] drop default profile view delete button
- [x] disable edit until implementation

Resolve #34

simply-alliv commented:
Clicking on the profile takes me directly to the profile edit page? Is that the expected behaviour?

piouson committed:
feat: add profile show page

- [x] change default profile page to show (previously edit)
- [x] remove profile edit page from view
- [x] remove edit button from profile show page
- [x] remove password field

Will revisit profile edit feature at a later date.

Resolve #35

piouson committed:
Merge remote-tracking branch 'upstream/develop' into profile-page

piouson committed:
chore: update dependencies
